tests/system/test_asyncpg_connection.py

```diff
@@ -32,6 +32,7 @@ async def create_sqlalchemy_engine(
     user: str,
     password: str,
     db: str,
+    ip_type: str,
```

Comment:
nit: let's default this to `"public"` so docs samples work
```suggestion
    ip_type: str = "public",
```